### DISCUSSION TITLE
Spawners won't spawn next mobs above MFM limit instead of despawning them after it

### DISCUSSION THREAD

**Discussion by bobhenl** [Zrips/CMI#3868] [Feature Requests]
**Description of feature/improvement request:**
---

### Detailed explanation of what it should do, when and in what situation: ###
Hi, now there's problem when players having mob traps and bunch of spawners, then when using MFM and having for ex. limit to monsters to 10, max 5 zombies, skeletons, creepers etc. then when they have for ex. 6 spawners and they're still spawning, limit is easily extended and mobs are despawned even they swim to player. So player can't kill any mobs and mob traps are broken. So compatibiltiy with MFM to stop CMI spawners to spawn mobs, when the limit is reached. To prevent despawning them... would be better to stop spawning instead of despawning them.
### Basic explanation as to why this feature is needed and/or useful to have: ###
It's needed for mob traps, because then they really don't work and players complain about it. Mobs are despawned earlier then they go to players through water. So it completely broken mob traps.

OR atleast despawn the latest mobs in MFM (disallow spawning another ones) instead of clearing all of them in chunk when the limit is reached

**Comment by CloudeLecaw** [comment]
Dont worry, they despawn on next check. You can define the time  in config. Pls use the MFM section next time
https://github.com/Zrips/MobFarmManager/issues

**Comment by bobhenl** [comment]
But I don't wanna let almost every of them despawn, I wanna just the newest one stop spawning.
